modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts

```diff
@@ -13,9 +13,41 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
     { label: 'Iframe Editor', setup: TinyHooks.bddSetup },
     { label: 'Shadow Dom Editor', setup: TinyHooks.bddSetupInShadowRoot }
   ], (tester) => {
+    context(`${tester.label}, Test of different color types`, () => {
+      const hook = tester.setup<Editor>({
+        base_url: '/project/tinymce/js/tinymce',
+        toolbar: 'forecolor backcolor',
+        content_style: 'p{color: pink; background-color: hsl(120, 100%, 75%);}'
+      }, []);
+
+      it('TINY-9213: Color detected on color, background', async () => {
```

Comment:
Should all the tests also close/cancel the dialog? Does the test end up with multiple dialogs open at the same time?